extensions/wikia/TemplateDraft/TemplateConverter.class.php

```diff
@@ -111,6 +111,27 @@ public function createDataTag( $variable ) {
 	}
 
 	/**
+	 * Overrides content of parent page with contents of draft page
+	 * @param Title $title Title object of sub page (draft)
+	 * @throws PermissionsException
+	 */
+	public function approveDraft( Title $title ) {
+		if ( $title->userCan( 'edit' ) ) {
```

Comment:
Should we instead check the permissions against the parent title? A user may be able to edit the draft, but the parent template may be protected, and this would enable them to edit the parent template when they don't have the appropriate permissions.